contracts/minting/MintController.sol

```diff
@@ -171,11 +189,11 @@ contract MintController is Controller {
         return internal_setMinterAllowance(minter, newAllowance);
     }
 
-   // Internal functions
-
     /**
      * @dev Uses the MinterManagementInterface to enable the minter and
```

Comment:
replace @dev with @notice

Reply:
Updated with your latest round of comments @mirathewhite

Reply:
Updated